### ISSUE TITLE
Increase user directory permissions

### ISSUE THREAD

**Issue by brittyazel** [flathub/chat.rocket.RocketChat#26]
Hi Folks,

So by default it looks like RocketChat only has permissions for Downloads and Pictures folders. Can we expand this to the whole user home directory? There's lots of files in Documents that I'm trying to share back and forth now that we're using Rocketchat heavily for work, with chat.gnome.org. And it's really annoying to have to move my files into downloads, to then be able to send in Rocketchat. 

Also, I know that I can overwrite the default perms, and that's fine for me, but it wasn't ok for Caroline who I just found out had been cursing Rocketchat under he breath thinking it was a bug lol.

TLDR: It would be nice for this and other productivity apps to have access to the whole USER dir.

**Comment by andrevmatos** [comment]
It was like that before, and #15 requested to move to a stricter security model. RocketChat doesn't need access to the whole home, which can contain quite sensible information.
Ideally, #11 should allow portal usage to expose any needed directory, but until them, we prefer to continue being more strict, and one should expose any directory needed on a per user basis:
https://askubuntu.com/questions/1086529/how-to-give-a-flatpak-app-access-to-a-directory
Thank you for your issue and interest, let us know if anything new arises.

**Comment by brittyazel** [comment]
That's understandable, I figured that was the rationale, however I really suggest you reconsider. RocketChat is a necessary part of the GNOME Foundation workflow, and sending files is part of that. It's somewhat non-ideal to have to instruct each employee to overwrite directory perms. Can you at least open up ~/Documents?

**Comment by andrevmatos** [comment]
@brittyazel It's reasonable. Done with #27

**Comment by brittyazel** [comment]
Thank you very much!

**Comment by andrevmatos** [comment]
I kept the specific folders shared in filesystem, but with #86, RocketChat uses portals by default, and will allow opening files outside of the sandbox.
